### ISSUE TITLE
Obsoletion request: GO:0007263 nitric oxide mediated signal transduction 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#26613]
Dear all, 

The proposal has been made to obsolete GO:0007263 nitric oxide mediated signal transduction. The reason for obsoletion is that the term has been inconsistently used, and not always to represent signaling. NO signaling goes via cGMP, so the correct term for nitric oxide signaling is GO:0038060 nitric oxide-cGMP-mediated signaling pathway.

**Comment by raymond91125** [comment]
GO:0007263 nitric oxide mediated signal transduction definition xref is PMID:21549190 which states that "a canonical pathway linked to cyclic GMP (cGMP) as its quintessential effector has been established, but other modes of action have emerged and are now part of the common body of knowledge within the field.". Do we still want to obsolete this term? @pgaudet

**Comment by pgaudet** [comment]
Yes, we want to obsolete that because there is a more specific term, [nitric oxide-cGMP-mediated signaling](http://purl.obolibrary.org/obo/GO_0038060)

However I don't know where annotation reviews stand. Suzi closed it as out of date but I dont think it was complete.

**Comment by mlkaldunski** [comment]
This paper indicates that nitric oxide mediated signal transduction can be cGMP independent. What would you suggest I use to annotate the Rasd1 gene actions, if all the non-cGMP terms are being obsoleted? intracellular signaling cassette (GO:0141124) seems awfully broad.
https://pubmed.ncbi.nlm.nih.gov/11086993/

**Comment by raymond91125** [comment]
> This paper indicates that nitric oxide mediated signal transduction can be cGMP independent. What would you suggest I use to annotate the Rasd1 gene actions, if all the non-cGMP terms are being obsoleted? intracellular signaling cassette (GO:0141124) seems awfully broad. https://pubmed.ncbi.nlm.nih.gov/11086993/

@pgaudet What's your take? Do we need a new term? Thanks.

**Comment by pgaudet** [comment]
Hi @mlkaldunski @raymond91125 Looking at https://pubmed.ncbi.nlm.nih.gov/11086993/, and a few other papers (https://pubmed.ncbi.nlm.nih.gov/16908409/, https://pubmed.ncbi.nlm.nih.gov/25129479/)

it seems this could be annotated to [NMDA selective glutamate receptor signaling pathway](http://purl.obolibrary.org/obo/GO_0098989)

Does that work ? 

Or perhaps we should create a calmodulin-nitric oxide signaling pathway, see https://pubmed.ncbi.nlm.nih.gov/41672893/? 
although this other paper seems to say that NO activates CAMK, and not the other way around: https://pubmed.ncbi.nlm.nih.gov/41667275/

What do you think ?

**Comment by mlkaldunski** [comment]
I can use the NMDA term, since the evidence for that data is in rat cortical neurons. It is a shame to lose the "nitric oxide mediated" part of the information though, so I may add a cellular response to nitric oxide annotation as well. Truth be told, I'd much rather just be able to keep the original term, it was perfect. I would not advocate creating a new term.
Mary

**Comment by raymond91125** [comment]
I think it makes sense to create a new term: nitric oxide-mediated cGMP-independent signaling
with definition: An intracellular signaling cassette that starts with production of nitric oxide and ends with the activation of downstream effectors that further transmit the signal within the cell. Nitric oxide transmits its downstream effects through cyclic GMP (cGMP)-independent mechanisms. [PMID:21549190]
There are several different mechanisms mentioned in the reference. When requested, we can add children later.

**Comment by pgaudet** [comment]
Let's try to avoid "independent"

How about 'protein S‐nitrosylation-mediated nitric oxide mediated signal transduction'? 

This seems to be the other 'major' mechanism of NO signaling - see PMID: [31347144](https://pubmed.ncbi.nlm.nih.gov/31347144/), and the paper you cite, @raymond91125  [PMID:21549190]

**Comment by pgaudet** [comment]
That being said, all papers I see about NMDA signaling seem to describe NO as an intrinsic part of the pathway; would this new term not create a double annotation ?

**Comment by raymond91125** [comment]
There are NMDA signaling independent of NO, e.g. PMID:38791034, PMID:12536131, PMID:11994750.

**Comment by pgaudet** [comment]
**PMID:38791034**
> NO Is Required for the LFS-Induced ACC-ACC Potentiation but Not for the TBS-Induced homo-LTP in the ACC

But they dont show that the TBS-Induced homo-LTP in the ACC pathway is via NMDA, do they? Looks like it's a different pathway altogether. 


**PMID:12536131** is a 2002 paper.  They focus on calcium, and since NMDA is a Ca ion channel, that is absolutely consistent with the upstream part of the pathway: (figure from PMID: [31347144](https://pubmed.ncbi.nlm.nih.gov/31347144/)

<img width="540" height="445" alt="Image" src="https://github.com/user-attachments/assets/970aef4c-2ca6-4125-b0a4-495f3d85a165" />

I dont see a mention of other downstream effectors. 

Same for **PMID:11994750** - they only mention the very early steps of the pathway, ie calcium import and calmodulin. 

The PSD95 adaptor seems key to link NO synthase, see PMID:30108200. Are there cases where NMDA is not coupled to PSD95?

**Comment by raymond91125** [comment]
NMDA signaling without nitric oxide

PMID:30501045 (2021) NMDA receptor -> internal Ca2+ release.

<img width="278" height="200" alt="Image" src="https://github.com/user-attachments/assets/b3dc5a44-2a68-4377-a299-113f4e917ff1" />

PMID: 34364898 (2021) NMDA receptors and synaptic plasticity in the anterior cingulate cortex

<img width="624" height="358" alt="Image" src="https://github.com/user-attachments/assets/63ddbe5e-4cf0-4265-a0d2-ab7f28459163" />

**Comment by LiNiMGI** [comment]
@pgaudet @raymond91125 Almost all the MGI annotations on the list are NOT cGMP-dependent, if we obsolete the GO:0007263 nitric oxide mediated signal transduction, will we create any new term for the cGMP-independent signaling pathway? thanks!

**Comment by raymond91125** [comment]
@dragon-ai-agent please obsolete the following terms, and replace with indicated GO terms.
GO:0007263 nitric oxide mediated signal transduction -> GO:0038060 nitric oxide-cGMP-mediated signaling
GO:0010749 regulation of nitric oxide mediated signal transduction -> GO:0141149 regulation of nitric oxide-cGMP mediated signal transduction
GO:0010751 negative regulation of nitric oxide mediated signal transduction -> GO:0141151 negative regulation of nitric oxide-cGMP mediated signal transduction
GO:0010750 positive regulation of nitric oxide mediated signal transduction -> GO:0141150 positive regulation of nitric oxide-cGMP mediated signal transduction

**Comment by dragon-ai-agent** [comment]
🤖 Working on it...

Follow along: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/23627111265)

*— @dragon-ai-agent*

**Comment by dragon-ai-agent** [comment]
The requested term obsoletions have been completed. I have created PR #31767 with the following changes:

### Terms Obsoleted

| Term ID | Original Name | Replaced By |
|---------|--------------|-------------|
| GO:0007263 | nitric oxide mediated signal transduction | GO:0038060 (nitric oxide-cGMP-mediated signaling) |
| GO:0010749 | regulation of nitric oxide mediated signal transduction | GO:0141149 (regulation of nitric oxide-cGMP mediated signal transduction) |
| GO:0010750 | positive regulation of nitric oxide mediated signal transduction | GO:0141150 (positive regulation of nitric oxide-cGMP mediated signal transduction) |
| GO:0010751 | negative regulation of nitric oxide mediated signal transduction | GO:0141151 (negative regulation of nitric oxide-cGMP mediated signal transduction) |

Each term has been properly marked as obsolete with:
- Name prefixed with "obsolete"
- Definition prefixed with "OBSOLETE."
- `is_obsolete: true` flag
- `replaced_by` pointing to the more specific cGMP-mediated term
- Link back to this issue via `term_tracker_item`

### Rewired References

Two terms that referenced the obsoleted terms were updated:
- **GO:0038060** (the replacement term): Now inherits from GO:0141124 (intracellular signaling cassette) and has the `has_participant CHEBI:16480` (nitric oxide) relationship
- **GO:1990658** (transnitrosylase complex): Now references GO:0038060 instead of GO:0007263

All validation checks pass. Ready for review.

---
🤖 **Generated by @dragon-ai-agent**
- Model: `claude-opus-4-5-20251101`
- Agent harness: claude-code
- Triggered by: @raymond91125
- Run: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/23627111265)
